lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -206,7 +206,33 @@ for node in $allocated_nodes; do
 done
 
 # Make sure all openstack services are down on all nodes before we stop rabbitmq
-sleep 10
+services_stopped=""
+count=0
+while [ "$services_stopped" != "yes" -a "$count" -lt 100 ]; do
+  echo "Waiting for OpenStack services to stop..."
+  sleep 1
+  services_stopped="yes"
+  for node in $allocated_nodes; do
+    if ssh "$node" 'shopt -s nullglob
+      for i in /etc/init.d/openstack-* ; do
+        if $i status >/dev/null ; then
+          echo "$i still running at `hostname`"; exit 0
+        fi
+      done
+      if pgrep -f neutron-ns-metadata-proxy >/dev/null; then
+        echo "neutron proxy is still runing at `hostname`"
+        exit 0
+      fi
```

Comment:
Actually, I'd remove that. If the killall didn't make the processes go away, then nothing will stop them. Or are we expecting that the killall will take a long time to take effect?